src/tree/localProject/ConfigTreeItem.ts

```diff
@@ -5,17 +5,24 @@
 
 import { AzExtTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
 import { treeUtils } from "../../utils/treeUtils";
-import { ConfigGroupTreeItem } from "./ConfigGroupTreeItem";
+import { BuildConfig, ConfigGroupTreeItem } from "./ConfigGroupTreeItem";
 
 export class ConfigTreeItem extends AzExtTreeItem {
     public static contextValue: string = 'azureStaticConfig';
     public contextValue: string = ConfigTreeItem.contextValue;
-    public readonly label: string;
+    public commandId: string = 'staticWebApps.openYAMLConfigFile';
+    public commandArgs: unknown[];
+    public readonly buildConfig: string;
     public parent: ConfigGroupTreeItem;
 
-    public constructor(parent: ConfigGroupTreeItem, label: string) {
+    public constructor(parent: ConfigGroupTreeItem, buildConfig: BuildConfig) {
         super(parent);
-        this.label = label;
+        this.buildConfig = buildConfig;
+        this.commandArgs = [this.parent.parent, this.buildConfig];
+    }
+
+    public get label(): string {
+        return this.buildConfig;
```

Comment:
I think if we use the yaml library, we could probably more easily display the value of the build config as well.

Reply:
I do think the yaml library will be needed for this part. The `parse` function gives us an object with all the yaml config